src/AccessibilityInsights.Extensions.AzureDevOps/AzureBoardsIssueReporting.cs

```diff
@@ -15,17 +15,51 @@ namespace AccessibilityInsights.Extensions.AzureDevOps
     [Export(typeof(IIssueReporting))]
     public class AzureBoardsIssueReporting : IIssueReporting
     {
-        private static AzureDevOpsIntegration AzureDevOps => AzureDevOpsIntegration.GetCurrentInstance();
+        private readonly FileIssueHelpers _fileIssueHelpers;
+        private readonly IDevOpsIntegration _devOpsIntegration;
+
+        /// <summary>
+        /// The one and only IDevOpsIntegration object used in production. It's static
```

Comment:
I'm not fully satisfied with this code. I'd much prefer to have a static setter on ConfigurationControl, but for reasons that elude me, it didn't want to resolve. I considered an event with a listener, but that also seemed like the wrong approach.